src/main/java/org/dice_research/opal/civet/metrics/DataFormatMetric.java

```diff
@@ -0,0 +1,124 @@
+package org.dice_research.opal.civet.metrics;
+
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.ModelFactory;
+import org.apache.jena.rdf.model.NodeIterator;
+import org.apache.jena.rdf.model.Property;
+import org.apache.jena.rdf.model.RDFNode;
+import org.apache.jena.rdf.model.Resource;
+import org.apache.jena.rdf.model.ResourceFactory;
+import org.apache.jena.rdf.model.SimpleSelector;
+import org.apache.jena.rdf.model.Statement;
+import org.apache.jena.rdf.model.StmtIterator;
+import org.apache.jena.vocabulary.DCAT;
+import org.apache.jena.vocabulary.DCTerms;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.dice_research.opal.civet.Metric;
+import org.dice_research.opal.common.vocabulary.Opal;
+
+
+/**
+ * The DataFormatMetric gives a rating to a dataset based on 
+ * whether a data-format or file-format information is available
+ * or not.
+ * 
+ * Datasets are available in the form of distributions where 
+ * each distribution is supposed to provide data/file format information
+ * through dct:format or dcat:mediaType predicate. In this metric the 
+ * validity of data/file format is not checked.
+ * 
+ * @see https://www.w3.org/TR/vocab-dcat-1/#Property:distribution_format
+ * @see https://www.w3.org/TR/vocab-dcat-1/#Property:distribution_media_type
+ * 
+ * @author Gourab Sahu
+ */
+public class DataFormatMetric implements Metric {
+
+	private static final Logger LOGGER = LogManager.getLogger();
+	private static final String DESCRIPTION = 
+			 " A dataset can have many distributions, we will calculate individual score for each distribution and"
+			 +"a distribution gets 5 star if it has non-empty value for property dcat:mediatype or dct:dataformat"
+			 + "We will not check the validity the provided mediatype(IANA type) or dataformat as this is not feasible "
```

Comment:
why is that not feasable?
the metric description will be shown to users. maybe rephrase (delete the sentence) it.